src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Query.cs

```diff
@@ -288,6 +288,11 @@ internal set
         /// </summary>
         public bool HasErrored { get; private set; }
 
+        /// <summary>
+        /// The Process ID of the query when connected.
+        /// </summary>
+        public string PID { get; private set; }
```

Comment:
nit: Agree with Alan that this should be updated to be either ServerProcessId (to match MDS) or ServerConnectionId (to match protocol)